tests/data/group_system/group_logging/group_ensure_rsyslog_log_file_configuration/rule_rsyslog_files_groupownership/IncludeConfig_is_other.fail.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+#
+# profiles = xccdf_org.ssgproject.content_profile_pci-dss
+# platform = Red Hat Enterprise Linux 7,Red Hat Enterprise Linux 8,multi_platform_fedora,multi_platform_ol
+
+# Check rsyslog.conf with root group-owner log from rules and
+# non root group-owner log from $IncludeConfig fails.
+
+RSYSLOG_CONF='/etc/rsyslog.conf'
+
+GROUP_TEST=testssg
+groupadd $GROUP_TEST
+
+GROUP_ROOT=root
+
+test_file=test1.log
+test_file_2=test2.log
+test_conf=test_conf.conf
+test_dir=/tmp/${0%.*}
+
+# setup test log files and ownership
+mkdir ${test_dir}
+
+touch ${test_dir}/${test_file}
+chgrp $GROUP_ROOT ${test_dir}/${test_file}
+
+touch ${test_dir}/${test_file_2}
+chgrp $GROUP_TEST ${test_dir}/${test_file_2}
+
+# backup default RSYSLOG_CONF file
+mv ${RSYSLOG_CONF} ${RSYSLOG_CONF}.bak
```

Comment:
Backup is not used in test at all. But creating backup is not needed, because test_suite creates snapshot, runs test and reverts machine to state before test, so you can delete this.